dotCMS/src/main/java/com/dotcms/rest/api/v1/maintenance/JVMInfoResource.java

```diff
@@ -34,7 +34,7 @@
 @SuppressWarnings("serial")
 public class JVMInfoResource implements Serializable {
 
-    final Pattern obfuscatePattern = Pattern.compile(Config.getStringProperty("OBFUSCATE_SYSTEM_ENVIRONMENTAL_VARIABLES", "/passw|pass|secret|key|pwd/gi"),Pattern.CASE_INSENSITIVE);
+    final Pattern obfuscatePattern = Pattern.compile(Config.getStringProperty("OBFUSCATE_SYSTEM_ENVIRONMENTAL_VARIABLES", "passw|pass|passwd|secret|key"),Pattern.CASE_INSENSITIVE);
```

Comment:
why this change? It seems that your branch is out of sync and this PR should be against release-21.09